server/player/player.go

```diff
@@ -1446,6 +1447,12 @@ func (p *Player) AttackEntity(e world.Entity) {
 		}
 		if vulnerable {
 			p.Exhaust(0.1)
+
+			if s, ok := i.Enchantment(enchantment.KnockBack{}); ok {
+				diff := float64(s.Level()) / 2
+				force += diff
+				height += diff
+			}
```

Comment:
Would maybe move these calculations into a method the enchantment itself.

Reply:
Changed.